quickstarts/dockerdesktop.md

```diff
@@ -0,0 +1,105 @@
+Docker Desktop comes includes a standalone Kubernetes server and client, designed for local testing of Kubernetes.
```

Comment:
`comes includes` sounds a bit weird ... a typo I guess?

Reply:
Yes, the word `comes` shouldn't be there